auth/service.go

```diff
@@ -632,7 +632,7 @@ func (svc service) RetrieveDomain(ctx context.Context, token, id string) (Domain
 		SubjectKind: UsersKind,
 		Object:      id,
 		ObjectType:  DomainType,
-		Permission:  MembershipPermission,
+		Permission:  ViewPermission,
```

Comment:
From the previous understanding of how the member work I thought he should not be able to login to a domain since he will see the other members' groups, things and channels

Reply:
`RetrieveDomain` function helps `member` to retrive present login domain information.
So this should have `MembershipPermission`